src/url-replacements.js

```diff
@@ -144,6 +148,82 @@ class UrlReplacements {
     this.set_('SCREEN_HEIGHT', () => {
       return this.win_.screen.height;
     });
+
+    // Returns the time it took to load the whole page. (excludes amp-* elements
+    // that are not rendered by the system yet.)
+    this.set_('PAGE_LOAD_TIME', () => {
```

Comment:
These definitely need to be documented in https://github.com/ampproject/amphtml/blob/master/spec/amp-var-substitutions.md and we likely need to introduce a table and/or subsections.